src/state/shell/logged-out.tsx

```diff
@@ -1,23 +1,43 @@
 import React from 'react'
 
-type StateContext = {
+const StateContext = React.createContext<{
   showLoggedOut: boolean
-}
-
-const StateContext = React.createContext<StateContext>({
+  requestedAccountSwitchTo?: string // did of account to switch to
```

Comment:
Nit: if we're gonna go with this approach, can you make these named arguments via an object? Just want it to be very clear 🙏